include/libtorrent/socket_type.hpp

```diff
@@ -56,7 +56,7 @@ enum class socket_type_t : std::uint8_t {
 };
 
 // return a short human readable name for types of socket
-char const* socket_type_name(socket_type_t);
+TORRENT_EXPORT char const* socket_type_name(socket_type_t);
```

Comment:
well, this is a very very nice to have :)

Reply:
it this should be public, maybe it should say it may return an empty string too

Reply:
done